rllib/examples/documentation/saving_and_loading_algos_and_policies.py

```diff
@@ -231,6 +231,7 @@ def new_policy_mapping_fn(agent_id, episode, worker, **kwargs):
 # ppo_config.framework("torch")
 
 # Create the Algorithm and train one iteration.
+# We disable the RLModules and Learner APIs here, since
```

Comment:
This comment is no applicable anymore I guess? the disabling happens further below.

Reply:
Whoops. Yeah wanted to go over these PRs before passing them into review lol. Thanks